src/bramble/client/main.js

```diff
@@ -920,6 +925,10 @@ define([
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_DISABLE_WORD_WRAP"}, callback);
     };
 
+    BrambleProxy.prototype.configureAutoCloseTags = function(options, callback) {
+        this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS", args: [ options ]}, callback);
```

Comment:
In Brackets, on the other side of the iframe, we consume the data here: https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/RemoteCommandHandler.js#L185-L189.  You can pass `args` as an array, or as a single item, and we'll create an array for you.  What you're doing here is the preferred way to do this, yes.